skorch/utils.py

```diff
@@ -128,10 +128,26 @@ def to_numpy(X):
 def to_device(X, device):
     """Generic function to move module output(s) to a device.
 
-    Deals with X being a torch tensor or a tuple of torch tensors.
+    Parameters
+    ----------
+    X : input data
+        Deals with X being a:
+        * torch tensor
```

Comment:
Can you add a newline before the bullet points for the docs to be properly formatted.

The first line of the docs should also be adjusted since it's not only about module outputs.

Reply:
Sure thing, I have updated the doc string. 👍